src/domain/Chain.js

```diff
@@ -0,0 +1,36 @@
+const config = require('@streamr/config')
+
+class Chain {
+	constructor(name /* string */) {
```

Comment:
here also jsdoc might work better: https://jsdoc.app/howto-es2015-classes.html

Reply:
...unless the idea of these comments is to turn this all into typescript ;)

Reply:
The reason for writing this in plain JS is to allow a wider audience of developers to contribute thus making it more accessible.